unic/utils/tests/macro_tests.rs

```diff
@@ -0,0 +1,56 @@
+#[macro_use]
+extern crate unic_utils;
+
+use unic_utils::char_property::EnumeratedCharProperty;
+
+char_property! {
+    pub enum Property {
+        /// Required
+        AbbrVariant {
+            abbr => AV,
+        }
+        /// Required
+        LongVariant {
+            abbr => LV,
+            long => Long_Variant,
+        }
+        /// Required
+        DisplayVariant {
+            abbr => DV,
+            display => "The one and only DISPLAY VARIANT!!!11!",
```

Comment:
nit: I know it's a test, but `!!!11!` doesn't test anything very specific, right? :D